src/release/breaking-changes/3-7-deprecations.md

```diff
@@ -0,0 +1,458 @@
+* [`ThemeData`][]
+* [`ColorScheme`][]
+
+Relevant issues:
+* [#56639][]
+* [#84748][]
+* [#56918][]
+* [#91772][]
+
+Relevant PRs:
+
+Deprecated in:
+* [#92822][]
+* [#81336][]
+* [#85144][]
+
+Removed in:
+* [#118658][]
+* [#119360][]
+* [#120577][]
+* [#120932][]
+
+[Accent color migration guide]: {{site.url}}/release/breaking-changes/theme-data-accent-properties
+[`ThemeData`]: {{site.api}}/flutter/widgets/Draggable-class.html
+[`ColorScheme`]: {{site.api}}/flutter/widgets/LongPressDraggable-class.html
+[#56639]: {{site.repo.flutter}}/pull/56639
+[#84748]: {{site.repo.flutter}}/pull/84748
+[#56918]: {{site.repo.flutter}}/pull/56918
+[#91772]: {{site.repo.flutter}}/pull/91772
+[#92822]: {{site.repo.flutter}}/pull/92822
+[#81336]: {{site.repo.flutter}}/pull/81336
+[#85144]: {{site.repo.flutter}}/pull/85144
+[#118658]: {{site.repo.flutter}}/pull/118658
+[#119360]: {{site.repo.flutter}}/pull/119360
+[#120577]: {{site.repo.flutter}}/pull/120577
+[#120932]: {{site.repo.flutter}}/pull/120932
+
+---
+
+### `AppBar`, `SliverAppBar`, and `AppBarTheme` updates
+
+Supported by Flutter Fix: yes
+
+In v2.4, several changes were made ot the app bar classes and their themes to
+better align with Material Design. Several properties were deprecated at that
+time and have been removed.
+
+For `AppBar`, `SliverAppBar` and `AppBarTheme`:
+* `brightness` has been removed, and is replaced by `systemOverlayStyle`
+* `textTheme` has been removed, and is replaced by either `toolbarTextStyle` or `titleTextStyle`.
+* `backwardsCompatibility` can be removed, as it was a temporary migration flag for these properties.
+
+Additionally, `AppBarTheme.color` was removed, with `AppBarTheme.backgroundColor`
+as its replacement.
+
+**Migration guide**
+
+Code before migration:
+```dart
+var toolbarTextStyle = TextStyle(...);
+var titleTextStyle = TextStyle(...);
+AppBar(
+  brightness: Brightness.light,
+  textTheme: TextTheme(
+    bodyMedium: toolbarTextStyle,
+    titleLarge: titleTextStyle,
+  )
+  backwardsCompatibility: true,
+);
+AppBarTheme(color: Colors.blue);
+```
+
+Code after migration:
+```dart
+var toolbarTextStyle = TextStyle(...);
+var titleTextStyle = TextStyle(...);
+AppBar(
+  systemOverlayStyle: SystemOverlayStyle(statusBarBrightness: Brightness.light),
+  toolbarTextStyle: toolbarTextStyle,
+  titleTextStyle: titleTextStyle,
+);
+AppBarTheme(backgroundColor: Colors.blue);
+```
+
+**References**
+
+API documentation:
+* [`AppBar`][]
+* [`SliverAppBar`][]
+* [`AppBarTheme`][]
+
+Relevant issues:
+* [#86127][]
+* [#70645][]
+* [#67921][]
+* [#67497][]
+* [#50606][]
+* [#51820][]
+* [#61618][]
+
+Deprecated in:
+* [#86198][]
+* [#71184][]
+
+Removed in:
+* [#120618][]
+* [#119253][]
+* [#120575][]
+
+
+[`AppBar`]: {{site.api}}/flutter/material/AppBar-class.html
+[`SliverAppBar`]: {{site.api}}/flutter/material/SliverAppBar-class.html
+[`AppBarTheme`]: {{site.api}}/flutter/material/AppBarTheme-class.html
+[#76739]: {{site.repo.flutter}}/pull/76739
```

Comment:
Seems this was intended to be [#86127](https://github.com/flutter/flutter/issues/86127) since that one (above) is missing a link definition and `#76739` isn't mentioned.

```suggestion
[#86127]: {{site.repo.flutter}}/pull/86127
```